gradle-plugin/build.gradle

```diff
@@ -62,7 +62,7 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
 
     // Because Gradle hardcodes Kotlin versions we have warnings during compilation. We might need
     // to disable this setting when upgrading to 1.5.0 stable.
-    allWarningsAsErrors = rootProject.ext.allWarningsAsErrors
+    allWarningsAsErrors = rootProject.ext.makeWarningsErrors
```

Comment:
Notice that somehow sharing the same variable name causes clashes. You gotta love Gradle / Groovy. Renaming the variable fixes the problem.